.github/pull_request_template.md

```diff
@@ -1 +0,0 @@
-- [ ] Changelog updated if needed
```

Comment:
@paultranvan : is it ok for you, to remove that?

I added it, after we notice together, one of your PR didn't update changelog.

Reply:
y'a un sujet en front à traiter sur ce changelog, faut absolument qu'on en parle mardi. En attendant on peut supprimer je pense.

Reply:
Why do you want to remove it? Do we still need to manually update the changelog?